### DISCUSSION TITLE
Help to unpack office suite

### DISCUSSION THREAD

**Discussion by apviveros** [minios-linux/minios-live#9] [Q&A]
Hi. I'm new here. I'm just a user. Don't have much knowledge other than downloading and using. I've just did so with this MiniOS. I need to install any office suite and downloaded WPS Office as .dev package but I can't unpack it to use it. I need help. Can you suggest anything?

**Comment by crim50n** [comment]
Unfortunately, WPS Office does not run in Debian 12 and in MiniOS. We are looking for a solution to this problem. I suggest you try OnlyOffice, it works well with Microsoft document formats.
